load_tests/load_test.py

```diff
@@ -9,8 +9,9 @@
 from datetime import datetime, timezone
 import create_testing_resources.kinesis_s3_firehose.resource_resolver as resource_resolver
 
-WAITER_SLEEP = 180
-MAX_WAITER_ATTEMPTS = 30
+WAITER_SLEEP = 300
```

Comment:
Increase sleep wait time so that it prints describe task output less often (I want the full output for debugging)